sensor/src/procedures/calibration.py

```diff
@@ -11,9 +11,11 @@ def __init__(
         self,
         config: custom_types.Config,
         hardware_interface: hardware.HardwareInterface,
+        simulate: bool = False,
     ) -> None:
         self.logger, self.config = utils.Logger(origin="calibration-procedure"), config
         self.hardware_interface = hardware_interface
+        self.simulate = simulate
```

Comment:
Is this utilized for the simulation?

Reply:
no, it's essentially a stub that can be used in the future to "simulate" the calibration step, but for now calibration is skipped entirely during simulation, so we could also remove this from calibration.py entirely